spec/spec_helper.rb

```diff
@@ -8,21 +9,21 @@
 require 'support/m2bf_xquery_helpers'
 require 'support/bib2lod_helpers'
   # Specific RDF entity helpers go here:
-require 'instances/instance_helpers'
-require 'publications/publication_helpers'
-require 'works/work_helpers'
+# require 'instances/instance_helpers'
+# require 'publications/publication_helpers'
+# require 'works/work_helpers'
```

Comment:
Go ahead and remove the lines if you don't want 'em -- the git history will still have 'em.  (I imagine this is just an artifact of your WIP anyway)